Public/Src/Engine/UnitTests/Scheduler/PipFingerprinterTests.cs

```diff
@@ -400,6 +401,94 @@ public void TestUserProfileStaysOutOfFingerprints()
             XAssert.IsTrue(json.Contains(Path.GetFileName(Environment.GetFolderPath(Environment.SpecialFolder.InternetCache)).ToUpperInvariant()));
         }
 
+        private ContentFingerprint GetWeakFingerPrintForExecutable(

```

Comment:
```suggestion
        private ContentFingerprint GetWeakFingerprint(
```
This method returns a fingerprint for the pip, not for the executable, hence the suggested change.